rich/style.py

```diff
@@ -345,7 +345,7 @@ def link(self) -> Optional[str]:
         return self._link
 
     @property
-    def transaprent_background(self) -> bool:
+    def transparent_background(self) -> bool:
```

Comment:
This one is technically a breaking change. Would you mind documenting this in  CHANGELOG.md

Reply:
Indeed breaking but I doubt anyone used it and failed to make a PR or at least raise a bug. Still, updating changelog seems reasonable.

Reply:
Done! FWIW, here's a full GitHub search for that token: https://github.com/search?q=transaprent_background&type=code 😅